pkg/operator/ceph/cluster/cluster.go

```diff
@@ -800,6 +800,9 @@ func initClusterCephxStatus(c *clusterd.Context, cluster *cephv1.CephCluster) er
 	cluster.Status.Cephx = &cephv1.ClusterCephxStatus{
 		RBDMirrorPeer: &uninitializedStatus,
 		Mgr:           &uninitializedStatus,
+		CSI: &cephv1.CephxStatusWithKeyCount{
+			CephxStatus: &uninitializedStatus,
+		},
```

Comment:
After changing the pointer, I anticipate this code will look something like this. I think this should work as expected.

```suggestion
		CSI: &cephv1.CephxStatusWithKeyCount{ // pointer is correct
			CephxStatus: uninitializedStatus, // change to non-pointer
		},
```

Reply:
done